applications/empire_application/python_scripts/empire_mapper.py

```diff
@@ -2,19 +2,17 @@
 # import libraries
 from KratosMultiphysics import *
 from KratosMultiphysics.EmpireApplication import *
-from KratosMultiphysics.IncompressibleFluidApplication import *
 import ctypes as ctp
 import os
 
 CheckForPreviousImport()
 
+path_of_the_application =  os.path.dirname((os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe()))))) # script directory
+# print(path_of_the_appli)
 # Import the mapper library
-# This is pre compiled and is put in the external library. Also a header file is put in the header file folder to give the 
-# users the syntax for the mapping routines. 
-#libMapper = ctp.cdll.LoadLibrary("../custom_external_libraries/libEMPIRE_MapperLib.so")
-libMapper = ctp.cdll.LoadLibrary("/home/annab/software/kratos/applications/empire_application/custom_external_libraries/libEMPIRE_MapperLib.so")
-
-#libMapper = ctp.cdll.LoadLibrary("libEMPIRE_MapperLib.so")
+# This is pre compiled and is put in the external library. Also a header file is put in the header file folder to give the
+# users the syntax for the mapping routines.
+libMapper = ctp.cdll.LoadLibrary(path_of_the_application+"/custom_external_libraries/libEMPIRE_MapperLib.so")

```

Comment:
From the point of view of Kratos, we would prefer not to distribute the dlls of Empire, because they are linux specific and because everything that is in the repository should be compatible with the kratos license (what is the license of Empire, btw?). Since you are already specifying a path to load the Empire so, would there be a way to specify an external path, and link to something that is distributed separately?
This would also have advantages regarding updates in Empire (you do not depend on kratos having the last version).

This is not something for this pull request, but what would be your opinion on this issue?